tests/lib/kubernetes/caasp.py

```diff
@@ -50,14 +51,16 @@ def __init__(self, workspace: Workspace, hardware: HardwareBase):
     def bootstrap(self):
         super().bootstrap()
         self.hardware.ansible_run_playbook('playbook_caasp.yaml')
-        self.workspace.execute("skuba cluster init --control-plane "
+        # Ansible retrieves the binary locally
+        os.chmod(self._skuba, 0o755)
+        self.workspace.execute(f"{self._skuba} cluster init --control-plane "
                                f"{self.hardware.masters[0].get_ssh_ip()} "
                                f"{self._clusterpath}", capture=True,
                                check=True)
 
-        logger.info("skuba node bootstrap. This may take a while")
+        logger.info(f"{self._skuba} node bootstrap. This may take a while")
```

Comment:
That looks wrong. Why do we need to know the binary path in the log here?

Reply:
Replacement error. Fixed.